### ISSUE TITLE
Switcher can't close and restart Steam when choosing another account

### ISSUE THREAD

**Issue by COMMANDERWONG** [TcNobo/TcNo-Acc-Switcher#112]
When I want to switch my steam account, the switcher can't close and restart steam with another account, steam just restart with the same account. Any ideas?

**Comment by TcNobo** [comment]
The Steam account switcher is the most stable and working well.
Try running the switcher as admin? Manually update by downloading a new copy?
And of course, make sure you have the correct Steam installation selected in the app. It could be editing the wrong files.
The file the switcher works with is: `C:\Program Files (x86)\Steam\config\loginusers.vdf`

**Comment by COMMANDERWONG** [comment]
I have to manually exit steam before I can use the switcher, the path is correct. Any ideas?

**Comment by TcNobo** [comment]
Does Steam not close at all when you try to switch accounts?
Try running the account switcher as admin, did that work?

**Comment by COMMANDERWONG** [comment]
The word "closing steam" is displayed for a second in the switcher but steam just reloads with my current without closing. Running as admin didn't solve this problem.

**Comment by COMMANDERWONG** [comment]
https://user-images.githubusercontent.com/37576365/145699228-d1318c04-56ee-460f-b527-7db3fb69700b.mp4

This is a demo of what happened when trying to switch my account

**Comment by TcNobo** [comment]
Download the latest release candidate from here (a beta version): https://github.com/TcNobo/TcNo-Acc-Switcher/releases/tag/2021-12-12_00
Let me know if this issue is fixed... It should be :)

In the Settings screen, you'll find the following toggle. Turn it on and try switching again, see if that helps:
![image](https://user-images.githubusercontent.com/10319195/145727272-d9648b3b-be68-4e48-900e-a1497cedfcf9.png)

**Comment by COMMANDERWONG** [comment]
Works like a charm, the issue is fixed, thank you! 
BTW, what did you change to make it work? I am curious.

**Comment by TcNobo** [comment]
https://github.com/TcNobo/TcNo-Acc-Switcher/commit/63428c2d0b6783876f17411332a584b7e0b41acd in TcNo-Acc-Switcher-Globals/Globals.cs 

The way I did it before was running CMD with `TASKKILL /F /T /IM <procname>`. This closed a bunch of processes at once very quickly, where closing them slowly one may restart others and nothing ends up happening. This avoided it well.

The alternate method is collecting a list of programs and then sending a Kill command, which works well... But I was doing it the other way as that seemed to work very quickly. They're probably equivalent tbh. Same with permissions.

-- This is probably the more 'standard' way of doing things come to think of it... The previous one I based off how I manually switched accounts with some apps.

---
Glad to know it's fixed. I'm closing this issue :)
